src/bin/clean.rs

```diff
@@ -0,0 +1,23 @@
+use std::io;
+use std::io::Write;
+use std::fs;
+use clap::{App, ArgMatches, SubCommand};
+use mdbook::MDBook;
+use mdbook::errors::Result;
+use get_book_dir;
+
+// Create clap subcommand arguments
+pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
+    SubCommand::with_name("clean").about("Delete built book")
+}
+
+// Init command implementation
+pub fn execute(args: &ArgMatches) -> Result<()> {
+    use mdbook::errors::*;
```

Comment:
It probably doesn't make sense to do a glob import here. Why don't you move it to the top of the file and replace the `use mdbook::errors::Result`?

Reply:
WIP: did.